### DISCUSSION TITLE
Connected, but not Syncing

### DISCUSSION THREAD

**Discussion by linoxsasha** [Chia-Network/chia-blockchain#9643] [Support]
![зображення](https://user-images.githubusercontent.com/96476422/146924757-6423423c-663f-4437-b485-66cd59f09ce6.png)

**Comment by dwp1064** [comment]
Hello, 
  I'm having this same problem, any updates?
Thank you,
David...

**Reply by dwp1064** [reply]
some additional info:

(venv) worker1@CHIA-FARM1:~/chia-blockchain/chia-blockchain-gui$ chia show -s
Exception from 'show' Traceback (most recent call last):
  File "/home/worker1/chia-blockchain/chia/cmds/show.py", line 41, in show_async
    blockchain_state = await client.get_blockchain_state()
  File "/home/worker1/chia-blockchain/chia/rpc/full_node_rpc_client.py", line 27, in get_blockchain_state
    response = await self.fetch("get_blockchain_state", {})
  File "/home/worker1/chia-blockchain/chia/rpc/rpc_client.py", line 49, in fetch
    raise ValueError(res_json)
ValueError: {'error': "b'\\xd7\\x80\\xd2,z\\x87\\xc9\\xe0\\x1d\\x98\\xb4\\x9a\\t\\x10\\xf6p\\x1c;\\x95\\x01WA1k?\\xda\\x04.]{\\x81\\xd2'", 'success': False}

**Reply by dwp1064** [reply]
Never mind, I killed the 40Gb main db and it started syncing from 0.

**Reply by Jacek-ghub** [reply]
Take a look at www.chia-database.com

**Reply by dwp1064** [reply]
I'll check it out, thank you!!
